byob.js

```diff
@@ -2,7 +2,7 @@
 
     byob.js
 
-    "build your own blocks" for SNAP!
+    "build your own blocks" for Snap!
```

Comment:
Can we maybe not touch every single JS file just to change the uppercase spelling of Snap in a comment? I'd rather not churn out new versions of every translation file, especially since we've been using the uppercase spelling a lot ourselves (and still do). Thanks!